addons/repair/stringtable.xml

```diff
@@ -1205,19 +1205,10 @@
             <Turkish>Sadece Gelişmiş Mühendis</Turkish>
         </Key>
         <Key ID="STR_ACE_Repair_engineerSetting_Wheel_name">
-            <English>Allow Wheel</English>
+            <English>Allow Wheel Removal</English>
```

Comment:
"Replacement" better fits the other translations. As this does both removal and "addition" right?